Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM node:10
+
+# Install package dependencies
+RUN DEBIAN_FRONTEND='noninteractive' apt-get -y update && \
+    DEBIAN_FRONTEND='noninteractive' apt-get -y upgrade
+RUN DEBIAN_FRONTEND='noninteractive' apt-get install -y --no-install-recommends netcat arp-scan
+
+RUN mkdir -p /app && \
+    mkdir -p /tmp
+
+WORKDIR /app
+
+COPY package.json ./
+
+RUN npm install
+COPY . .
+RUN npm run-script build
```

Comment:
This actually isn't necessary. `npm run start` serves an unbuilt version of the React app and Node server (e.g. it watches for changes to the React app and serves those changes as they happen).

I haven't figured out a clean way to serve a build version of the React app and Node server. If we find that `npm run start` is just too slow on the Pi, we can work that out and it _should_ run faster.